tests/common/models.py

```diff
@@ -396,3 +396,15 @@ def configure_tiny_bert_config():
 
 def configure_tiny_bert_hf_model(use_logits=True):
     return HuggingFaceModel(configure_tiny_bert_model(), configure_tiny_bert_tokenizer(), use_logits)
+
+
+def configure_tiny_gpt_model():
+    return copy.deepcopy(pytest.tiny_gpt_model)
```

Comment:
oops sorry, can you add the `try/excepts` that the bert ones have? I missed that in my PR

Reply:
Yep, added.